README.md

```diff
@@ -180,7 +180,6 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 
 ## Docker
 - :video_camera: [kickstart docker](https://www.youtube.com/watch?v=gAkwW2tuIqE)
-- :video_camera: [Complete Docker Guide](https://www.youtube.com/watch?v=pTFZFxd4hOI)
  
```

Comment:
Hey, it seems you deleted one of the `Docker` resource from line 183.

Reply:
I don't remember this but might have been deleted by mistake. Sorry for this.